Source/Coinbase/CoinbaseApiAuthenticator.cs

```diff
@@ -57,11 +58,15 @@ public void Authenticate(IRestClient client, IRestRequest request)
             var method = request.Method.ToString().ToUpper(CultureInfo.InvariantCulture);
 
             var body = string.Empty;
-
-            var param = request.Parameters.FirstOrDefault(p => p.Type == ParameterType.RequestBody);
-            if (param != null && param?.Value?.ToString() != "null" && !string.IsNullOrWhiteSpace(param?.Value?.ToString()))
-                body = param.Value.ToString();
-
+			if (request.Method != Method.GET) {
```

Comment:
Okay, no problem! Just wanted to double check. Thanks again for your contribution! One last question: Were you able to verify the code changes worked against sandbox/production?